suite-native/device/src/middlewares/deviceConnectionMiddleware.ts

```diff
@@ -161,7 +161,7 @@ deviceConnectionMiddleware.startListening({
         const isDeviceRemembered =
             !!device.features && selectDevices(getOriginalState()).some(d => d.id === device.id);
 
-        if (isDeviceRemembered) return;
+        if (isDeviceRemembered && selectIsCoinEnablingInitFinished(getState())) return;
```

Comment:
Please extract `selectIsCoinEnablingInitFinished(getState())` into a constant, it's also used on line 168.

Reply:
Fixed